speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/rppUtils/RppUtils.hpp

```diff
@@ -7,6 +7,17 @@
 #define _SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING 1
 #include <experimental/filesystem>
 
+// If compiling for a Windows system, the compiler will define one of these values.
+// Define the OS_WINDOWS property so we can easily check if we're running on Windows elsewhere,
+// and include Windows headers
+#if defined(WIN32) || defined(_WIN32) || defined(__WIN32) && !defined(__CYGWIN__)
```

Comment:
This is beyond the scope of this PR, portability will be added later and should not be done now. The way that we are going to support cross-platform has not been set yet, but this is also not the correct way to do it in code. So please remove this and make sure it works on Windows only for now